Fix authn-policy task: add missing cleanup and a dependent lib (#4410)
istio/istio.io: content/docs/tasks/security/authn-policy/index.md

$ wget {{< github_file >}}/security/tools/jwt/samples/key.pem
{{< /text >}}

> You may also need to download the [jwcrypto](https://pypi.org/project/jwcrypto) library,

Reviewer: ```suggestion
{{< tip >}}
Download the [jwcrypto](https://pypi.org/project/jwcrypto) library,
```

Author: Done. By the way, what's the most straightforward way to preview the markdown with all of the macros?

> if it is not already present on your system.

Reviewer: ```suggestion
if you haven't installed it on your system.
{{< /tip >}}
```
